Update endpoint slices docs (#38174)
kubernetes/website: content/en/docs/concepts/services-networking/endpoint-slices.md

at different times.

{{< note >}}
Clients of the EndpointSlice API must iterate through all the existing EndpointSlices

> associated to a Service and build a complete list of unique network endpoints. It is

Reviewer: nit

```suggestion
associated with a Service and build a complete list of unique network endpoints. It is
```

> associated to a Service and build a complete list of unique network endpoints. It is
> important to mention that endpoints may be duplicated in different EndointSlices.

Reviewer: ```suggestion
associated to a Service and build a complete list of unique network endpoints. Endpoints might be duplicated in different EndointSlices.
```

Reviewer: Done ☺️
